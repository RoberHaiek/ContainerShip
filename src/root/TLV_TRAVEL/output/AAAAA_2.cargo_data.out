container10,unload,0,0,3
container5,unload,0,0,3
container1,unload,0,0,3