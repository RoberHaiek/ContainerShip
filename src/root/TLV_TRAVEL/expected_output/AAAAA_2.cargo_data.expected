container1,unload,1,1,1
container5,unload,1,1,1
container10,unload,1,1,1